code/modules/mob/living/simple_animal/distortion/plague/bunnyman.dm

```diff
@@ -45,6 +45,12 @@
 	var/can_act = TRUE
 	var/transformed = FALSE
 
+//Set random stats for the egoist
```

Comment:
After that's done, you can remove this, and just change the egoist_attributes variables